modulemd/v2/include/modulemd-2.0/modulemd-component-module.h

```diff
@@ -92,4 +92,18 @@ modulemd_component_module_set_repository (ModulemdComponentModule *self,
 const gchar *
 modulemd_component_module_get_repository (ModulemdComponentModule *self);
 
+
+/**
+ * modulemd_component_module_equals:
```

Comment:
Don't make this a public function. It shouldn't be called directly. We expect consumers to use `modulemd_component_equals()` instead. This should be a static function defined only in `modulemd-component-module.c`

Reply:
done